agent/bench-scripts/pbench-uperf

```diff
@@ -549,6 +557,10 @@ for protocol in `echo $protocols | sed -e s/,/" "/g`; do
 									if [ -z "$client" ]; then
 										client="127.0.0.1"
 									fi
+                                    uperf_server=`echo $uperf_servers | cut -d, -f$server_nr`
+                                    if [ -e "$uperf_server" ]; then
+                                        uperf_server="$server"
```

Comment:
Is `-e` the correct test?  (I.e., is there possibly a _file_ somewhere with this name??)  Or should this be `-z`?

Reply:
Definitely not, will fix this as well.